docker-compose-base.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
```

Comment:
How is this base file used? I don't see it included in the other files.

Reply:
As stated in the README, you concatenate multiple compose files, for instance: `docker-compose -f docker-compose-base.yml -f docker-compose-s64da-cpu.yml up`.

Reply:
Oh, sorry, I missed that.